src/redux/reducers/ConceptReducers.js

```diff
@@ -1,3 +1,4 @@
+// import uuid from 'uuid';
```

Comment:
What is the above used for?

Reply:
@dkayiwa I have taken out the commented out code.